src/renderer/screens/exchange/Swap2/Form/index.js

```diff
@@ -61,12 +61,16 @@ const SwapForm = () => {
   const { providers, error: providersError } = useSwapProviders();
   const storedProviders = useSelector(providersSelector);
   const exchangeRate = useSelector(rateSelector);
+  const setExchangeRate = useCallback(
+    rate => {
+      dispatch(updateRateAction(rate));
+    },
+    [dispatch],
+  );
```

Comment:
Let's discuss next week what we plan to do regarding the redux store.

Reply:
Yeah, using the redux store could even be totally avoided if I reuse the trick to set the `<Drawer />` state from the outside (to be double checked).